### PR TITLE
Add support for advanced numbers like 6 to 9

### PR DESCRIPTION
**Pull request by jdegger**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -30,6 +30,10 @@ export const Divide = (props) => (
 
 export const One = () => <span>1</span>
 export const Two = () => <Add><One/><One/></Add>
-export const Three = () => <Add><One/><Two/></Add>
+export const Three = () => <Add><Two/><One/></Add>
```
Comment (SaschaMann) [inline index.js]: #2 already fixes this.

Comment (jdegger) [reply index.js]: Reverted that

**Review by btnwtn — APPROVED** [review]
I like these advanced numbers, great PR.

**Comment by jdegger** [comment]
Fair warning: don't use them if you're not incredibly good at real math

**Pull request merged**